Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageView.swift

```diff
@@ -263,8 +264,6 @@ public struct LinkDetectionTextView: View {
         LocalizedStringKey(message.adjustedText)
     }
     
-    @State var displayedText: AttributedString?
```

Comment:
We should be careful here of breaking changes around markdown links, will try to find the threads so we can test.

Reply:
From my texts, all works good. At least regular texts 🤔 Maybe this was related to edge cases?

Reply:
For example this one: https://github.com/GetStream/stream-chat-swiftui/pull/582, works fine

Reply:
I shared one example on slack, let's check that one as well.

Reply:
I've also tested that one, works fine